database/config.js

```diff
@@ -34,10 +35,4 @@ module.exports = {
     min: DB_MIN_POOL ? parseInt(DB_MIN_POOL, 10) : 1,
     max: DB_MAX_POOL ? parseInt(DB_MAX_POOL, 10) : 10,
   },
-
-  // Migrations and seeders options
-  migrationStorage: "sequelize",
```

Comment:
Why remove this? Do we have this elsewhere?

Reply:
no, we don't. this arose as a result of types actually. let me briefly outline the sequence of steps below for clarity: 

1. conversion to `.ts`
2. used `SequelizeOptions` from `sequelize-typescript`
3. type error thrown on the removed properties as they **do not** exist on the `SequelizeOptions` type.

This then implies that these properties don't exist or should not be set, which is why i removed it.